src/ButtonType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+/**
+ * Types for the button component.
+ *
+ * @see https://getbootstrap.com/docs/5.2/components/buttons/#examples
+ */
+enum ButtonType: string
```

Comment:
Button type is `submit`, `reset` or `button`. Suggest rename this enumeration to `ButtonDesign` or `ButtonAppearance`.

Reply:
I would like to have a standard for all widgets, if the name is changed it should be changed in the Alert, let's ask @samdark who is very good with names.

Reply:
Bootstrap calls it "variant".

[Buttons](https://getbootstrap.com/docs/5.3/components/buttons/#variants):

```
Bootstrap includes several button **variants**, each serving its own semantic purpose, with a few extras thrown in for more control.
```

[Alerts](https://getbootstrap.com/docs/5.3/components/alerts/#examples):

```
Heads up! As of v5.3.0, the alert-**variant**() Sass mixin is deprecated. Alert **variants** now have their CSS variables overridden in a Sass loop.
```

May be call enum `ButtonVariant`?

Reply:
done.